ecs-cli/modules/cli/compose/container/container_test.go

```diff
@@ -126,6 +127,15 @@ func TestPortString(t *testing.T) {
 	}
 }
 
+func TestHealthStatus(t *testing.T) {
+	containerHealth := "SOME_VAL"
+	container := setupContainer()
+	container.ecsContainer.SetHealthStatus(containerHealth)
+
+	// container.HealthStatus() should simply report the value returned by the ECS API
+	assert.Equal(t, containerHealth, container.HealthStatus())
+}
+
```

Comment:
This test seems a little frivolous but all the other ones also seem similar so I guess it's 🆗